galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/ResultNames.java

```diff
@@ -54,8 +58,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 
 		Collections.sort(resultsList);
 
-		if(!ExtractQuerySort.isAscending(queryParams, "resultname")) {
-			Collections.reverse(resultsList);
+		try {
+			if (!ExtractQuerySort.isAscending(queryParams, "resultname")) {
```

Comment:
what is a resultname? Not heard of that before. How does it differ from a runname ?